Game/Hymn.json

```diff
@@ -9,7 +9,7 @@
 			"z" : 1.0
 		},
 		"dither" : false,
-		"fog" : true,
+		"fog" : false,
```

Comment:
I don't think we want to turn off the fog.

Reply:
changed to true